index.bs

```diff
@@ -1498,6 +1527,30 @@ The [=public key credential=] type uses certain data structures that are specifi
 follows.
 
 
+### User Verification Requirement enumeration (enum <dfn enum>UserVerificationRequirement</dfn>) ### {#userVerificationRequirement}
+
+<pre class="idl">
+    enum UserVerificationRequirement {
+        "required",
+        "wanted",
```

Comment:
I like "preferred". How about "discouraged" for the last one?

I feel there's a value in having the options in order of the level of requirement. Ascending or descending doesn't matter, but I think "preferred" belongs in the middle since it's the middle level of requirement.

Reply:
I still like "Not required" better as discouraged is something prescriptive and looks like Platform has to choice while asking. "Not required" says that platform MUST not ask for it even if authenticator supports it. 

I will leave it up to you.

Reply:
I think "discouraged" better communicates that the RP has an *active* opinion that the client should not ask for UV, while "not required" instead communicates that the RP has no specific opinion other than that UV is passively allowed but not required. "not preferred" also feels too passive to me, and "forbidden" is too strict since some authenticators may always require UV.